src/Cms/App.php

```diff
@@ -169,6 +177,7 @@ public function api(): Api
 
     /**
      *  Apply a hook to the given value
+     * @internal
```

Comment:
Why is the collection method internal as well?

Reply:
That's a mistake